Detections/AuditLogs/PrivlegedRoleAssignedOutsidePIM.yaml

```diff
@@ -21,7 +21,7 @@ tags:
 query: |
   AuditLogs
   | where Category =~ "RoleManagement"
-  | where ActivityDisplayName =~ 'Add member to role (permanent)'
+  | where ActivityDisplayName =~ 'Add member to role'
```

Comment:
This detection is related to adding the role outside of PIM and specifically associated with this content - https://docs.microsoft.com/en-us/azure/active-directory/fundamentals/security-operations-privileged-accounts#things-to-monitor-1.

It looks like there may be a more specific Activity that aligns with this now - 'Add member to role outside of PIM (permanent)' but need to get confirmation.  @ashwin-patil - can you please have a further look at this when you get back next week.